src/components/NewLocationPage/SparkLines/SingleSparkLine.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { scaleLinear } from '@vx/scale';
+import { LinePath } from '@vx/shape';
+import { curveNatural } from '@vx/curve';
+import { COLOR_MAP } from 'common/colors';
+import { Point } from './utils';
+
+const SingleSparkLine: React.FC<{
+  data: Point[];
+  chartWidth: number;
+  chartHeight: number;
+  capY: number;
+}> = ({ data, chartWidth, chartHeight, capY }) => {
+  const yScale = scaleLinear({
+    domain: [0, capY],
+    range: [chartHeight, 0],
+  });
+
+  const xScale = scaleLinear({
+    domain: [0, 29],
+    range: [0, chartWidth],
+  });
+
+  const getXCoord = (p: Point) => xScale(p.x);
+  const getYCoord = (p: Point) => yScale(p.y);
```

Comment:
Maybe we don't need these functions, they make a bit more sense when the calculation is more involved or when we need to reuse them a couple of times.